projects/rulenode-core-config/src/lib/components/transform/copy-keys-config.component.html

```diff
@@ -1,28 +1,27 @@
-<section [formGroup]="copyKeysConfigForm" fxLayout="column">
-  <div class="mat-caption" style="margin-bottom: 8px">{{'tb.rulenode.copy-from' | translate}}</div>
-  <mat-radio-group formControlName="fromMetadata" fxLayout="row" fxLayoutGap="16px">
-    <mat-radio-button [value]="false" color="primary" >
-      <span>{{'tb.rulenode.data-to-metadata' | translate}}</span>
-    </mat-radio-button>
-    <mat-radio-button [value]="true" color="primary">
-      <span>{{'tb.rulenode.metadata-to-data' | translate}}</span>
-    </mat-radio-button>
-  </mat-radio-group>
-  <mat-form-field class="mat-block" subscriptSizing="dynamic">
-    <mat-chip-grid formControlName="keys" #attributeChipList>
+<section [formGroup]="copyKeysConfigForm" fxLayoutGap="16px">
```

Comment:
Class tb-form-panel should be used instead of attribute fxLayoutGap="16px"

Reply:
Done